controllers/telegram_handlers.py

```diff
@@ -79,6 +84,25 @@
 """
 
 
+def create_calendar_markup():
+    today = datetime.date.today()
+    year = today.year
+    month = today.month
+    dates = []
+    global list_dates
+    list_dates = []
+
+    first_day = datetime.date(year, month, 1)
+    last_day = datetime.date(year, month + 1, 1) - datetime.timedelta(days=1)
+
+    for day in range(1, last_day.day + 1):
+        date = datetime.date(year, month, day)
+        dates.append(InlineKeyboardButton(str(day), callback_data=date.strftime("%Y-%m-%d")))
+        list_dates.append(date.strftime("%Y-%m-%d"))
+
+    return InlineKeyboardMarkup([dates[i:i + 7] for i in range(0, len(dates), 7)])   
```

Comment:
Move this to utils and call it as a function. its cluttering up `telegram_handlers.py`